### PR TITLE
ubuntu trusty to focal

### PR DESCRIPTION
**Pull request by dobriseb**
Hi,
Here is my proposition of update to the last versions of ubuntu and harfbuzz

### CODE REVIEW

**Comment by NyakudyaA** [comment]
LGTM @dobriseb have you tested this. 

Don’t you need to also configure the https://github.com/kartoza/docker-mapserver/blob/master/resources/php5-fpm.conf

File: Dockerfile [hunk 1]
```diff
@@ -41,20 +43,20 @@ RUN /setup.sh
...
-RUN  apt-get install -y libapache2-mod-fastcgi php5-fpm libapache2-mod-php5 php5-common php5-cli php5-fpm php5
+# Install PHP7.4 and necessary modules
+RUN  apt-get install -y php7.4-fpm libapache2-mod-php7.4 php7.4-common php7.4-cli php7.4-fpm php7.4 #libapache2-mod-fastcgi doesn't exist anymore
 
 # Enable these Apache modules
 RUN  a2enmod actions cgi alias
 
-# Apache configuration for PHP-FPM
-RUN cp /tmp/resources/php5-fpm.conf /etc/apache2/conf-available/
+# Apache configuration for PHP-FPM # No fastcgi anymore
```
Comment (NyakudyaA) [inline Dockerfile:58]: How about the following https://www.interserver.net/tips/kb/apache-php-fpm-ubuntu-18-04/

Comment (dobriseb) [reply Dockerfile:58]: Yes, I've tested it and it's working for me.
I forgot to comment line 57 cp php5-fpm.conf
Focal is 20-04 and I haven't found apache2 fastcgi in packages.

**Comment by NyakudyaA** [comment]
Thanks @dobriseb

**Pull request merged**